client/src/components/pages/Home/ReportBuilding/Form/index.js

```diff
@@ -0,0 +1,221 @@
+import React, { Component } from 'react';
+import { Steps, notification } from 'antd';
+import PropTypes from 'prop-types';
+import axios from 'axios';
+
+import FirstStep from './FirstStep';
+import SecondStep from './SecondStep';
+import ThirdStep from './ThirdStep';
+import styles from './form.module.css';
+
+const { Step } = Steps;
+
+const steps = [
+  {
+    title: 'Empty Building',
+  },
+  {
+    title: 'Extra Information',
+  },
+  {
+    title: 'Personal Information',
+  },
+];
+
+class Form extends Component {
+  state = {
+    current: 0,
+    stepOneValues: {
+      city: '',
+      address: '',
+      previousUse: '',
+      owner: 'N/A',
+      isOwnerLocal: '',
+    },
+    stepTwoValues: {
+      emptyPeriod: '',
+      extraInfo: '',
+      preferredUse: '',
+      thumbnail: '',
+    },
+    stepThreeValues: {
+      reporterName: '',
+      reporterEmail: '',
+      reporterAddress: '',
+      receiveNotifications: false,
+      shareData: false,
+    },
+  };
+
+  getStepOneValues = values => {
+    const { stepOneValues } = this.state;
+    this.setState({
+      stepOneValues: {
+        ...stepOneValues,
+        ...values,
+      },
+    });
+  };
+
+  getStepTwoValues = values => {
+    const { stepTwoValues } = this.state;
+    this.setState({
+      stepTwoValues: {
+        ...stepTwoValues,
+        ...values,
+      },
+    });
+  };
+
+  getStepThreeValues = values => {
+    const { stepThreeValues } = this.state;
+    this.setState({
+      stepThreeValues: {
+        ...stepThreeValues,
+        ...values,
+      },
+    });
+  };
+
+  handleConfirm = values => {
+    const { stepThreeValues } = this.state;
+    this.setState(
+      {
+        stepThreeValues: {
+          ...stepThreeValues,
+          ...values,
+        },
+      },
+      () => this.sendData(),
+    );
+  };
+
+  sendData = async () => {
+    const openNotificationWithIcon = (type, message) => {
+      notification[type]({
+        message,
+        duration: 3,
+      });
+    };
+    const {
+      stepOneValues,
+      stepTwoValues: { emptyPeriod, extraInfo, preferredUse, thumbnail },
+      stepThreeValues,
+    } = this.state;
+
+    const { longitude, latitude, redirectToView } = this.props;
+
+    const formData = new FormData();
+    const building = {
+      ...stepOneValues,
+      longitude,
+      latitude,
+      emptyPeriod,
+      extraInfo,
+      preferredUse,
+      ...stepThreeValues,
+    };
+
+    formData.append('data', JSON.stringify(building));
+    formData.append('thumbnail', thumbnail[0].originFileObj);
```

Comment:
```suggestion
    if(thumbnail && thumbnail[0])
        formData.append('thumbnail', thumbnail[0].originFileObj);
```

Reply:
I did it but forget to push it 
I will push it ^_^
@ahmedisam99